app/src/controllers/plugins/uiExtensions/createImportProps.js

```diff
@@ -35,7 +35,7 @@ import { BigButton } from 'components/buttons/bigButton';
 import { NavigationTabs } from 'components/main/navigationTabs';
 import { NoCasesBlock } from 'components/main/noCasesBlock';
 import { ItemList } from 'components/main/itemList';
-import { ModalLayout, ModalField } from 'components/main/modal';
```

Comment:
The old `ModalLayout` component is also used in other plugins, so until a redesign for them, we need to leave this component exposed here.